### ISSUE TITLE
"Basic HTML version" does not offer access to all content info as "Standard version" does

### ISSUE THREAD

**Issue by dorina1** [wet-boew/GCWeb#1254]
When we select the “Basic HTML version” link, all the dynamic widgets functionality is not available anymore, and it is expected to have the same access to all the elements available in the “Standard version”.

When using IE11, on the [Canada.ca](https://www.canada.ca/en.html) website with the "Basic HTML version" link selected, I cannot access with JAWS, keyboard-only or mouse-only any of the links available under heading level 2 Topic Menu. JAWS will read the top menu links as “same page links” but there is no functionality.

Plus (as already reported) we cannot get all of the content in a slideshow/carousel as headings and content. Only 1 slide out of 4 slides (image and link to content info) is available for users using the Basic HTML version.
 
@wet-boew/access-working-group 
![basichtmlversion](https://cloud.githubusercontent.com/assets/1918938/26105236/2f083caa-3a0f-11e7-953a-9cea6a8830f5.jpg)

**Comment by LaurentGoderre** [comment]
This is an implementation issue. These should be links to a landing page that provide the same functionality as the megamenu

/cc @lucas-hay @masterbee

**Comment by rlambert27** [comment]
My opinion is that we should remove the "Basic HTML version", i've yet to see concrete evidence that it's useful. Only very vague references to "we need to support older AT tools", although i've seen no mention of what these old AT tools are and why they don't work.

It's like when back in the day we wanted WET to support ALL existing browsers, it seems to me like the link does the same but for AT tools.

**Comment by kodecount** [comment]
@rlambert27 Their was all ready a long discussion about this that lead to https://github.com/wet-boew/wet-boew/pull/7899. Also a lot of issues @dorina1 brought up are fixed now it just up to whoever is responsible for their domain to update to the latest WET.

**Comment by kodecount** [comment]
> This spring, we did testing across desktop and mobile of Canada.ca and the consultation section. We looked at all the major AT tools (only last 2-3 versions) in all 4 categories of disability technology (vision, hearing, physical, ld/cognitive) using windows, mac, ios and android. During our testing many of the issues that users were running into were able to be surmounted by enabling the basic html mode. - @mysticjeff

**Comment by LaurentGoderre** [comment]
Even if the AWG didn't come this conclusion, it would be too big of a change for a minor release to remove.

**Comment by RobJohnston** [comment]
And the broken links in basic mode were reported in https://github.com/wet-boew/GCWeb/issues/1163, back in August.

**Comment by RobJohnston** [comment]
I think that the single item in the carousel was fixed by https://github.com/wet-boew/wet-boew/pull/7815, as reported in https://github.com/wet-boew/GCWeb/issues/1081.  Should be available for the next release... [any day now](http://wet-boew.github.io/wet-boew/docs/versions/rdmp-en.html).

That just leaves the implementation of the menu.

**Comment by EricDunsworth** [comment]
@lucas-hay @RobJohnston Should this be closed? This issue, #1163 and #1324 cover the same menu problem in Canada.ca' implementation of the GCWeb theme. I noticed both of you recently closed the other two issues for that reason.

@lucas-hay Do you happen to know if there's been any progress in fixing the Canada.ca' menu? AFAIK that issue has been present since Canada.ca was migrated to the MWS late in the summer of 2016.

Btw I believe the carousel problem mentioned in the OP was resolved in wet-boew/wet-boew#7815. Prior to that PR, only the first carousel panel would appear in noscript, wbidsable and print views. That PR modified the carousel's SCSS styles to make all panels visible in those views.

**Comment by RobJohnston** [comment]
Yeah, I closed #1163 because @lucas-hay closed #1324, and because you opened a request in the Principal Publisher's ticketing system for Canada.ca

**Comment by lucas-hay** [comment]
@EricDunsworth, @masterbee is working on it for canada.ca. I do not know when it is going to be deployed.

**Comment by EricDunsworth** [comment]
@lucas-hay @RobJohnston But given that #1163 and #1324 were closed and this issue is a duplicate of them, shouldn't this be closed as well?

Btw my intent isn't to downplay the menu problem identified in this issue. IMO noscript, wbdisable and print views should function as well as they possibly can, across the board and without any exceptions. But if a decision has been made to close any GCWeb issues related to implementation problems of that theme on Canada.ca, this issue would technically fall within the scope of that.

**Comment by bsouster** [comment]
@dorina1 Is this still an issue? 

From what I can tell, the megamenu now goes to the landing pages that include all the pages normally navigated via megamenu drop-down, and the carousel is fully available.

Let me know and I will fix it

**Comment by dorina1** [comment]
@bsouster  - the issue is almost fixed. Almost all of the megamenu items are working properly when in Basic HTML version except for the Travel menu item (it switches back to standard HTML version with the carousel enabled) and Health menu item (where the social media containers are not adjusting properly - the containers stretch under the carousels items).  At least this is the behaviour when the page is accessed with IE 11 or Edge.

**Comment by dfarough** [comment]
I just posted a related issue with regard to the share widget.
the share page link does not appear when a user activates the basic html option for a page.

https://github.com/wet-boew/wet-boew/issues/8475
